src/Faithlife.Utility.Dapper/Faithlife.Utility.Dapper.csproj

```diff
@@ -25,8 +25,19 @@
     <PackageReference Include="Dapper" Version="1.50.2" />
     <PackageReference Include="SourceLink.Create.GitHub" Version="2.0.2" PrivateAssets="all" />
     <PackageReference Include="SourceLink.Test" Version="2.0.2" PrivateAssets="all" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
+    <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
     <DotNetCliToolReference Include="dotnet-sourcelink" Version="2.0.2" />
     <DotNetCliToolReference Include="dotnet-sourcelink-git" Version="2.0.2" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
+    <PackageReference Include="System.Data.SqlClient">
+      <Version>4.8.6</Version>
+    </PackageReference>
```

Comment:
Why add a reference to `System.Data.SqlClient` to this package?

This package only depends on Dapper, not any particular ADO.NET provider.